tests/rptest/utils/si_utils.py

```diff
@@ -210,8 +210,10 @@ def get_ntp_sizes(fdata_per_host, hosts_can_vary=True):
             orig_ntp_size -= size_overrides[ntp]
         assert ntp in restored_ntps, f"NTP {ntp} is missing in the restored data"
         rest_ntp_size = restored_ntps[ntp]
-        assert rest_ntp_size <= orig_ntp_size, \
-            f"NTP {ntp} the restored partition is larger {rest_ntp_size} than the original one {orig_ntp_size}."
+        fraction = float(max(rest_ntp_size, orig_ntp_size)) / float(
+            min(rest_ntp_size, orig_ntp_size))
+        assert fraction < 1.1, \
+            f"NTP {ntp} the size of the restored partition is too different {rest_ntp_size} compared to the original one {orig_ntp_size}."
```

Comment:
Just making sure I understand this (and might be worth adding as a comment), is the change here that we now have the full-fidelity segments (same size) _plus_ new archival snapshots or somesuch? Wondering why we need the 1.1 fudging

Reply:
the size on disk can be different due to fallocation strategy